build_darwin.sh

```diff
@@ -57,15 +57,15 @@ gpg tor-$TOR_VERSION.tar.gz.asc
 echo "$TOR_HASH  tor-$TOR_VERSION.tar.gz" | shasum -a 256 -c - && \
 tar -xvzf tor-$TOR_VERSION.tar.gz
 cd tor-$TOR_VERSION && \
-./configure --prefix=$PWD/install \
+./configure --prefix=$PWD/bin \
             --enable-static-libevent \
             --enable-static-openssl  \
             --with-libevent-dir=$PWD/../libevent-$LIBEVENT_VERSION/install \
-            --with-openssl-dir=$PWD/../openssl-$OPENSSL_VERSION/ \
+            --with-openssl-dir=$PWD/../openssl-$OPENSSL_VERSION/lib \
             --disable-asciidoc \
             ac_cv_func_getentropy=no \
             ac_cv_func_clock_gettime=no && \
-make ${jobs:+-j${jobs}} && make ${jobs:+-j${jobs}} check
+make ${jobs:+-j${jobs}} && make ${jobs:+-j${jobs}} check && make install
 cd ..
 
-cp tor-$TOR_VERSION/src/app/tor tor-$TOR_VERSION-darwin-brave-$BRAVE_TOR_VERSION
+cp tor-$TOR_VERSION/bin/tor tor-$TOR_VERSION-darwin-brave-$BRAVE_TOR_VERSION
```

Comment:
Errr...  If you set prefix to $PWD/bin, shouldn't this be $TOR_VERSION/bin/bin/tor?

Reply:
(this is why I suggested something like `root` or `installroot` or `usr` or `prefix`, so it wouldn't be confused with an existing directory name like `bin` or `lib`)